e107_plugins/pm/pm_shortcodes.php

```diff
@@ -575,13 +575,13 @@ public function sc_pm_delete($parm = '')
 			}
 
 
-			if($parm != '')
+			if(count($parm) == 0)
```

Comment:
Thanks. Would you be okay to update it? I'm trying to use 'empty' throughout the core where applicable.